views/js/controller/inspectResults.js

```diff
@@ -105,6 +106,24 @@ define([
                     taskCreationData : {uri : uniqueValue}
                 }).start();
             });
+
+            binder.register('proctorio_url_redirect', function(actionContext) {
+                const data = _.pick(actionContext, ['uri', 'classUri', 'id']);
+
+                request(this.url, {...data},'POST')
```

Comment:
Why do you need `{...data}`?
```suggestion
                request(this.url, data} ,'POST')
```

Reply:
Overlooked, fixed.